modules/serverless-rn-1-2-0.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * serverless/release-notes.adoc
+
+[id="serverless-rn-1-2-0_{context}"]
+//update the <version> to match the filename
+
+= Release Notes for Red Hat {ServerlessProductName} Technology Preview 1.2.0
+// add a version, e.g. Technology Preview 1.0.0
+
+[id="new-features-1-2-0_{context}"]
+== New features
+* {ServerlessProductName} has been updated to use Knative Serving 0.9.0.
+* {ServerlessProductName} on {product-title} 4.2 now uses the Operator Lifecycle Manager dependency resolution mechanism to install the ServiceMesh operator automatically.
+* Access to the KnativeServing resource is now restricted to `cluster-admin` roles to prevent any user from blocking the resource.
+* The {ServerlessProductName} Operator can now be found in the OperatorHub by searching for "knative".
+* The {product-name} web console now shows status conditions for the KnativeServing resource.
+
+
+[id="fixed-issues-1-2-0_{context}"]
+== Fixed issues
+* In previous releases, using the same services or routes in different namespaces caused services to not work properly and caused {product-title} routes to be overriden. This issue has been fixed.
+* In previous releases, different traffic split targets required a mandatory tag. A single traffic split can now be defined with `untagged` traffic targets.
+
+[id="known-issues-1-2-0_{context}"]
+== Known issues
+* Users may see the error `Unknown Uninitialized : Waiting for VirtualService to be ready` while waiting for a service.
```

Comment:
A comment on this has been moved to the "Getting started" guide. Is that good enough?

Reply:
Probably - can you point me to where that is? Thanks.

Reply:
It's in your own PR for the docs regarding Service Mesh.